pyuvsim/utils.py

```diff
@@ -261,7 +261,10 @@ def get_avail_memory():
     using psutils methods.
     """
     if not HAVE_PSUTIL:
-        raise ImportError("You need psutils to run this function. Install pyuvsim[sim]")
+        raise ImportError("You need psutils to estimate available memory. "
+                          "Install it by running pip install pyuvsim[sim] "
+                          "or pip install pyuvsim[all] if you also want mpi4py, "
+                          "h5oy, and line_profiler installed.")
```

Comment:
I think mpi4py will get installed if you use the [sim], so that's extra here. Also `h5oy` should be `h5py`